lib/ts/Layout.cc

```diff
@@ -41,56 +47,61 @@ Layout::get()
 }
 
 void
-Layout::create(const char *prefix)
+Layout::create(ts::string_view prefix)
```

Comment:
I'm not sure that I like passing `string_view` by value everywhere. I think that `const ts::string_view&` makes it clearer to the reader that this is effectively a const parameter. @dragon512 what do you think?